.circleci/config.yml

```diff
@@ -147,19 +148,19 @@ jobs:
       # build to use only the required Python files.
       - run:
           name: Build Docker image
-          command: docker build -t app:build --build-arg PIP_FILE=build.txt .
+          command: docker build -t app:prod --target=base .
```

Comment:
It occurs to me that we could actually restore the production image here, now that we're building it. Not a big deal though, this can wait until a subsequent PR.

Reply:
I can modify this to load the image instead of rebuilding, it should just be a small change.